siip.tex

```diff
@@ -305,13 +315,14 @@
           Co-PI Davis will be invited to engage in these video conference as 
           part of action research toward the final 
           As progress is made, lessons learned will be recorded by all 
-          participants in a draft instruction manual. 
+          participants in a draft instruction manual.
+          % TODO: what of Sp18?
```

Comment:
Sp18 is before the second workshop. Thus, still part of the interim period.  I don't understand this question.